site/src/load.rs

```diff
@@ -213,6 +215,51 @@ impl SiteCtxt {
         )
     }
 
+    /// Returns the not yet tested published artifacts, sorted from newest to oldest.
+    pub async fn missing_artifacts(&self) -> anyhow::Result<Vec<String>> {
+        let artifact_list: String = reqwest::get("https://static.rust-lang.org/manifests.txt")
+            .await?
+            .text()
+            .await?;
+
+        lazy_static! {
+            static ref VERSION_REGEX: Regex = Regex::new(r"(\d+\.\d+.\d+)").unwrap();
+        }
+
```

Comment:
```suggestion
```

Reply:
I don't understand this comment, the suggestion looks like I should delete these lines?

Reply:
Sorry for not being clear. I believe this statis is not being used so it can be deleted.

Reply:
Oh, of course! :facepalm: My bad, sorry.